contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,166 @@
+/>
+</BorderWrapper>
+
+When prioritizing your roadmap, you don't need to overthink things. Stack-ranking your projects by assigning each one a label of `low/medium/high` for effort and impact is sufficient (or you can even [add dollar estimates](/product-engineers/fixing-growth-problems#problem-2-prioritization)). This will give you a clear idea of which projects to work on first.
+
+### Step 5: Repeat
+
+Keep repeating the process from steps 1-4. Each time you do, you'll learn more about what you need to improve and how to prioritize it.
+
+As you start shipping your improvements, continue tracking your PMF survey score. It should begin to improve as you gradually convert your on-the-fence users into loyal advocates. 
+
+Lastly, to keep your product team focused on a single goal, consider making your PMF score their primary OKR.
+
+
+### Frequently Asked Questions
+
+#### How many responses do I need?
+
+Ellis recommends a minimum of 30 responses for the survey to become directionally useful. However, he also mentions that 100 or more responses is ideal.
+
+#### What do I do about users who answer "not disappointed"?
+
+Respectfully ignore them! 
+
+It's likely that there are many aspects of your app that are bothering or not benefiting them. Thus, they are essentially a lost cause. This is why you should instead focus on the "somewhat disappointed" users mentioned in step 3.
+
+## Case study: Superhuman
+
+Superhuman [nearly doubled their PMF score](https://review.firstround.com/how-superhuman-built-an-engine-to-find-product-market-fit) from 33% to 58%. More importantly, users became noticeably more vocal about how much they loved the product. Here's how they did it:
+
+1. **Segmented users:** They assigned user personas to everyone who responded (e.g., founder, engineer, customer success) and created an [Ideal Customer Profile](/founders/creating-ideal-customer-profile) based on the users that responded "very disappointed". 
+
+2. **Doubled down on what their ideal customers love:** People loved how fast Superhuman was and how quickly it helped them get their work done. Superhuman made sure to continue improving on this. They built more features to increase email efficiency – such as "Snippets": the ability to automatically type phrases, paragraphs, or whole emails. 
+
+3. **Converted on-the-fence users:** They focused on the users who loved Superhuman's speed but still responded with "somewhat disappointed". They learned that these users need a mobile app, and so they built it. Their users loved them for it!
+
+4. **Repeated the process:** Superhuman continued to survey users, tracking progress towards the 40% mark. The score became the primary OKR for the product team and, after three quarters, Superhuman increased their score to 58%.
+
+## Case study: Slack
```

Comment:
Do we have a case study that actually used it? If not, I think we should drop these.

1. The Superhuman case study is weaved throughout the main article, and could maybe be done more. 
2. Only including Superhuman and Slack makes the PMF survey feel not useful, only the team who invented it actually used it. Slack didn't even use it, it was just run by someone. It feels like proof it isn't worth doing that no one has done one or had results.